src/content/resources/resources.mdx

```diff
@@ -172,11 +172,11 @@ title: Resources
 
 ### IBM Plex
 
-IBM uses the font [Plex](https://github.com/IBM/plex) across products for brand consistency and cohesion.
+Carbon uses the open-source typeface [IBM Plex](https://github.com/IBM/plex) – carefully designed to meet IBM's needs as a global technology company and reflect IBM's spirit, beliefs, and design principles.
 
 ### IBM Design Language
 
-Carbon uses the [IBM Design Language](https://www.ibm.com/design/language/) as its guiding principle.
+Carbon delivers the [IBM Design Language](https://www.ibm.com/design/language/) as tooling for designers and developers, education, outreach, and support.
```

Comment:
This is a little confusing. How do you feel about:

"Carbon delivers the IBM Design Language through code and design tooling. The system also offers guidance, education, outreach, and support."

Reply:
trying to reinforce our broader messaging around design systems: 
>A design system is a Design language delivered as tooling for designers and developers, outreach, and activation.